testing/cli/test_cli.py

```diff
@@ -1,8 +1,8 @@
 import pytest
-import pkg_resources
 import importlib
+from importlib.metadata import entry_points
 
-scripts = pkg_resources.get_entry_map('spinalcordtoolbox')['console_scripts'].keys()
+scripts = [cs.name for cs in entry_points()['console_scripts'] if cs.name.startswith("sct_")]
```

Comment:
Note: As far as I can tell, there's no official `importlib.metadata` way to filter `'console_scripts'` by package; `entry_points()` just lists _all_ of them. So, I figured filtering by `'sct_'` would be distinctive enough to fetch only SCT's scripts.

Reply:
Actually, you know what, there's a way to be a little more explicit here...